### ISSUE TITLE
The menu of the MorphoLibJ is a little muddled

### ISSUE THREAD

**Issue by kld123509945** [ijpb/MorphoLibJ#22]
According to the reading sequence on the introduction (https://imagej.net/MorphoLibJ), I gradually understand the function of this great plug-in, though the introduction has a different sequence than the menu in the plug-in. 

As a inconvenient result, e.g., I found that when I'm using the "Max inscribed circle" function, there's always only one particle to be analysed in my binary image, which in fact has two particles. I have read the introduction carefully once again. Then finally I realised that I should apply the "connected components labelling" in advance to label the binary image so that I could do further analysis. How could the "connected components labelling" to stay at the second place? This trick hide sooooo inconspicuous....

Thus I think the menu sequence of the plug-in is a little muddled, which are telling us to analysis the image at first. In addition, some function should be attributed into one sub-menu, e.g. Gray Scale Attribute Filtering. I really don't understand how important this function is that it is placed in the main menu of MorphoLibJ....In addition, the introduction is not very friendly.  For instance, I notice that the section of "Binary and label image utilities" in the introduction is placed at the end. Thanks to its position that I almost didn't look at it....Introducing image processing order in alphabetical order is terrible.

However, all these inconvenience cannot stop the usage of this great plug-in for the people who really need it. So if there's really some difficulty to fix these, please ignore this thread. Many thanks!

**Comment by dlegland** [comment]
Hello,

thank you for your interest in the MorphoLibJ library, and for your feedback!

It is complicated to find adequate ordering of items in a menu. From many discussions with experts and novices in image processing, each person has its own "ideal order" for a menu organization, depending on its background and knowledge on the field. Moreover, the importance of the items totally depend on the type of image procesing workflow...

We have tried to create meaningful submenus, grouped by type of operations. We also tried to minimize the quantity of plugins or items in a given menu. The actual organization is also result of historical development, and is still likely to change. The user-manual is also a work-in-progress, and to my knowedge few other ImageJ plugins propose very detailed manuals. Also, the original purpose of the library was to propose a collection of plugins, not a complete introduction to image processing.

I understand some choices may seem strange. We continuously try to improve the library and the manual, and will try to include some of your remarks in future versions.

Best,
David
